libraries/Microsoft.Bot.Schema/SemanticAction.cs

```diff
@@ -33,10 +33,12 @@ public SemanticAction()
         /// </summary>
         /// <param name="id">ID of this action</param>
         /// <param name="entities">Entities associated with this action</param>
-        public SemanticAction(string id = default(string), IDictionary<string, Entity> entities = default(IDictionary<string, Entity>))
+        /// <param name="state">State of this action. Allowed values: `start`, `continue`, `done`</param>
+        public SemanticAction(string id = default(string), IDictionary<string, Entity> entities = default(IDictionary<string, Entity>), string state = default(string))
```

Comment:
Also can't do this. 

Have to have an additional ctor where "state" is required. (aka: overloads are ok)

Reply:
removed the state parameter